src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  let ACTION_REQUIRED = '';
```

Comment:
The variable ACTION_REQUIRED should be in camel case (actionRequired) according to JavaScript naming conventions.